addons/sourcemod/scripting/vsh/abilities/ability_brave_jump.sp

```diff
@@ -143,7 +157,16 @@ methodmap CBraveJump < SaxtonHaleBase
 			
 			float vecAng[3];
 			GetClientEyeAngles(this.iClient, vecAng);
-			if ((vecAng[0] < -25.0) && (this.iJumpCharge > 1))
+			
+			//Cap value to prevent impossible angle
+			float flAngleRequirement = this.flEyeAngleRequirement;
+			if (flAngleRequirement > 90.0)
+				flAngleRequirement = 90.0;
+				
+			else if (flAngleRequirement < -90.0)
+				flAngleRequirement = -90.0;
```

Comment:
Can move checks to `CBraveJump.flEyeAngleRequirement.set` when setting value, without needing to cap on every button release